test/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is also only necessary for DefinitelyTyped, so you can remove this.

Reply:
removed this one